ANMN/LTSP/TSaggregator/aggregated_timeseries.py

```diff
@@ -218,22 +310,27 @@ def main_aggregator(files_to_agg, var_to_agg):
     variable_attributes = set_variableattr(varlist, variable_attributes_templatefile)
 
     ## build the output file
-    nc_aggr = xr.Dataset({var_to_agg:                       (['OBS'],variableMainDF['VAR'].astype('float32'), variable_attributes[var_to_agg]),
-                          var_to_agg + '_quality_control':  (['OBS'],variableMainDF['VARqc'].astype(np.byte), variable_attributes[var_to_agg+'_quality_control']),
-                          'TIME':                           (['OBS'],variableMainDF['TIME'], variable_attributes['TIME']),
-                          'DEPTH':                          (['OBS'],variableMainDF['DEPTH'].astype('float32'), variable_attributes['DEPTH']),
-                          'DEPTH_quality_control':          (['OBS'],variableMainDF['DEPTH_quality_control'].astype(np.byte), variable_attributes['DEPTH_quality_control']),
-                          'instrument_index':               (['OBS'],variableMainDF['instrument_index'].astype('int64'), variable_attributes['instrument_index']),
+    nc_aggr = xr.Dataset({var_to_agg:                       (['OBSERVATION'],variableMainDF['VAR'].astype('float32'), variable_attributes[var_to_agg]),
```

Comment:
I don't know why even if I define the dtype when I create the empty DF, I have to specify it again. I will test if I remove from the creation step it will work

Reply:
Ok, no worries, I was just wondering if it was necessary.